dex/testing/polygon/harness.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+
+SESSION="polygon-harness"
+
+SOURCE_DIR=$(pwd)
+NODES_ROOT=~/dextest/polygon
+GENESIS_JSON_FILE_LOCATION="${SOURCE_DIR}/genesis.json"
+HARNESS_DIR=${NODES_ROOT}/harness-ctl
+
+mkdir -p "${NODES_ROOT}/${NAME}"
+mkdir -p "${NODES_ROOT}/beta"
+mkdir -p "${HARNESS_DIR}"
+
+NAME="alpha"
+PASSWORD="abc"
+
+cat > "${HARNESS_DIR}/${NAME}" <<EOF
+#!/usr/bin/env bash
+bor \$*
+EOF
+chmod +x "${HARNESS_DIR}/${NAME}"
+
+# Shutdown script
+cat > "${HARNESS_DIR}/quit" <<EOF
+#!/usr/bin/env bash
+tmux send-keys -t $SESSION:1 C-c
+# tmux send-keys -t $SESSION:2 C-c
+tmux kill-session
+EOF
+chmod +x "${HARNESS_DIR}/quit"
+
+GROUP_DIR="${NODES_ROOT}/${NAME}"
+NODE_DIR="${GROUP_DIR}/node"
+mkdir -p "${NODE_DIR}"
+
+ALPHA_ADDRESS="18d65fb8d60c1199bb1ad381be47aa692b482605"
+ALPHA_ADDRESS_JSON_FILE_NAME="UTC--2021-01-28T08-47-02.993754951Z--18d65fb8d60c1199bb1ad381be47aa692b482605"
+ALPHA_ADDRESS_JSON='{"address":"18d65fb8d60c1199bb1ad381be47aa692b482605","crypto":{"cipher":"aes-128-ctr","ciphertext":"927bc2432492fc4bbe9acfe0042f5cd2cef25aff251ac1fb2f420ee85e3b6ee4","cipherparams":{"iv":"89e7333535aed5284abd52f841d30c95"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"6fe29ea59d166989be533da62d79802a6b0cef26a9766fa363c7a4bb4c263b5f"},"mac":"c7e2b6c4538c373b2c4e0be7b343db618d39cc68fa872909059357ff36743ca0"},"id":"0e2b9cef-d659-4a26-8739-879129ed0b63","version":3}'
+ALPHA_NODE_KEY="71d810d39333296b518c846a3e49eca55f998fd7994998bb3e5048567f2f073c"
+ALPHA_ENODE="897c84f6e4f18195413c1d02927e6a4093f5e7574b52bdec6f20844c4f1f6dd3f16036a9e600bd8681ab50fd8dd144df4a6ba9dd8722bb578a86aaa8222c964f"
+ALPHA_NODE_PORT="10563"
+ALPHA_AUTHRPC_PORT="24331"
+ALPHA_HTTP_PORT="38556"
+ALPHA_WS_PORT="38557"
```

Comment:
These need to be changed. They are the same as the Ethereum harness.

Reply:
okay